README.md

```diff
@@ -31,22 +31,26 @@ Each argument is prepended with "`--`".
-* The source maven repository has to provide an "index-like" html view of your repository (successfully tested with Nexus2).
+* The source maven repository has to provide an "index-like" html view of your repository (successfully tested with Nexus2 and Nexus3).
 * The destination repository must accept HTTP PUT requests for storing maven artifacts and accept HTTP Basic authentication (successfully tested with Nexus3).
 
 ### Configuration
 The command-line arguments are as follows (arguments are prepended with "`--`"):
 
 Argument        | Mandatory | Description
-----------------|-----------|---------------
-source.root-url | Yes | The root URL from which the source repository should be scraped.
-source.user     | No  | The username used on the source repository (default: no authentication).
-source.password | No  | The password used on the source repository (default: no authentication).
-mirror-path     | No  | The path on the local file system to be used for mirroring repository content (default: `./mirror/`)
-target.root-url | Yes | The root URL where the content of the mirror repository should be uploaded.
-target.user     | No  | The username used on the target repository (default: no authentication).
-target.password | No  | The password used on the target repository (default: no authentication).
-actions         | No  | Comma-separated list of actions to be performed.<br/>Possible values:<ul><li>**`mirror`:** copy content from the source repository to the `mirror-path` on the local filesystem</li><li>**`publish`:** upload content from the `mirror-path` on the local filesystem to the target repository</li></ul>(default: `mirror,publish`)
+----------------|----------|---------------
+source.root-url          | Yes      | The root URL from which the source repository should be scraped.
+source.user              | No       | The username used on the source repository (default: no authentication).
+source.password          | No       | The password used on the source repository (default: no authentication).
+source.download-interval | No       | The interval between downloads, value in milliseconds, used to reduce load on source-server (default: 1000) 
+mirror-path              | No       | The path on the local file system to be used for mirroring repository content (default: `./mirror/`)
+target.root-url          | Yes      | The root URL where the content of the mirror repository should be uploaded.
+target.user              | No       | The username used on the target repository (default: no authentication).
+target.password          | No       | The password used on the target repository (default: no authentication).
+target.upload-interval   | No       | The interval between uploads, value in milliseconds, used to reduce load on target-server (default: 1000)
+target.concurrent-uploads| No       | Number of parallel uploads, parallelism is only supported on same folder, like foo/bar/artifact/1.0.0/ (default: 2)
```

Comment:
This might be nitpicking but one might ask: What is the reason for adding concurrent uploading but not concurrent downloading?

Reply:
in my scenario, download is fron an old server who does'nt have a high bandwith, but upload is to a new server who benefits from parallelism, and with idempotency, high amount of head requests could be done concurrently when the processes is interrupted and restarted, until get a valid new upload

Reply:
Ok I see, thanks for the clarification. :-)